testlib/testResources.go

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+package testlib
+
+import (
+	"fmt"
```

Comment:
Can you organize these imports ala what `goimports` would do?

Namely, stdlib first, then empty line, then the `mattermost-server` imports?